docs/src/components/ExternalExampleLayout.tsx

```diff
@@ -26,23 +27,29 @@ const ExternalExampleLayout: any = props => {
   if (!examplePath) return <PageNotFound />
   const exampleSource: ExampleSource = exampleSourcesContext(examplePath)
 
+  const isRtlEnabled = rtl === 'true'
+
   return (
-    <SourceRender
-      babelConfig={babelConfig}
-      source={exampleSource.js}
-      renderHtml={false}
-      resolver={importResolver}
-    >
-      <SourceRender.Consumer>
-        {({ element, error }) => (
-          <>
-            {element}
-            {/* This block allows to see issues with examples as visual regressions. */}
-            {error && <div style={{ fontSize: '5rem', color: 'red' }}>{error.toString()}</div>}
-          </>
-        )}
-      </SourceRender.Consumer>
-    </SourceRender>
+    <Provider theme={{ rtl: isRtlEnabled }}>
+      {/*<div dir={isRtlEnabled ? 'rtl' : undefined}>*/}
```

Comment:
Not sure whether we should add this wrapper div that will have dir='rtl' if rtl is enabled. We have that logic in the ComponentExample component.

Reply:
We definitely need it.

### Without

![image](https://user-images.githubusercontent.com/14183168/51974897-a8a4c180-2489-11e9-84cd-31161978ccc3.png)


### With

![image](https://user-images.githubusercontent.com/14183168/51974922-b8bca100-2489-11e9-99d5-fd6fa0114803.png)